docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  cinema:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app

```

Comment:
Mounting the entire current directory into the container (`.:/app`) can lead to potential issues if there are files that should not be included in the container. Ensure your `.dockerignore` file is properly configured to exclude unnecessary files.